deploy/acm-policies/50-GENERATED-srep-vap-vcpu-overcommit.Policy.yaml

```diff
@@ -0,0 +1,178 @@
+                                  name: windows-vcpu-overcommit
+                            matchConstraints:
+                                resourceRules:
+                                    - apiGroups:
+                                        - kubevirt.io
+                                      apiVersions:
+                                        - '*'
+                                      operations:
+                                        - CREATE
+                                        - UPDATE
+                                      resources:
+                                        - virtualmachineinstances
+                            validations:
+                                - expression: |-
+                                    (
+                                      'kubevirt.io/cluster-preference-name' in object.metadata.annotations &&
+                                      object.metadata.annotations['kubevirt.io/cluster-preference-name'].lowerAscii().contains('dedicated')
+                                    ) ||
+                                    (
+                                      'kubevirt.io/preference-name' in object.metadata.annotations &&
+                                      object.metadata.annotations['kubevirt.io/preference-name'].lowerAscii().contains('dedicated')
+                                    )
+                                  message: Windows VM are required to use *dedicated preferences.
+                    - complianceType: mustonlyhave
+                      metadataComplianceType: musthave
+                      objectDefinition:
+                        apiVersion: admissionregistration.k8s.io/v1
+                        kind: ValidatingAdmissionPolicyBinding
+                        metadata:
+                            name: windows-vcpu-overcommit-binding
+                        spec:
+                            policyName: windows-vcpu-overcommit
+                            validationActions:
+                                - Deny
+                pruneObjectBehavior: DeleteIfCreated
+                remediationAction: enforce
+                severity: low
+    remediationAction: enforce
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
+metadata:
+    name: placement-srep-vap-vcpu-overcommit
+    namespace: openshift-acm-policies
+spec:
+    clusterSelector:
+        matchExpressions:
+            - key: hypershift.open-cluster-management.io/hosted-cluster
+              operator: In
+              values:
+                - "true"
+            - key: openshiftVersion-major-minor
+              operator: NotIn
+              values:
```

Comment:
This excludes `4.14-4.18` but doesn't specify which versions it includes? meaning it also includes `4.13-` which is very old version already EOL?